templates/ts-scrape-single-page/package.json

```diff
@@ -1,20 +1,25 @@
 {
-    "name": "getting-started-typescript",
+    "name": "ts-scrape-single-page",
     "version": "0.0.1",
     "type": "module",
     "description": "This is an example of an Apify actor.",
+    "engines": {
+        "node": ">=18.0.0"
+    },
     "dependencies": {
-        "apify": "^3.0.0"
+        "apify": "^3.1.4",
+        "axios": "^1.4.0",
+        "cheerio": "^1.0.0-rc.12"
     },
     "devDependencies": {
         "@apify/tsconfig": "^0.1.0",
         "ts-node": "^10.9.1",
-        "typescript": "^4.9.5"
+        "typescript": "^5.0.4"
     },
     "scripts": {
         "start": "npm run start:dev",
         "start:prod": "node dist/main.js",
-        "start:dev": "ts-node-esm -T src/main.ts",
+        "start:dev": "ts-node-esm -T main.ts",
```

Comment:
Better to keep the `src` folder. Currently, the tsconfig will only include files from `src`, so `main.ts` in root will fail most probably (maybe not via ts-node, but during compilation)

Reply:
Perhaps we could just include the `main.ts` instead of the src in `tsconfig`. We wanted to keep the structure consistent with other templates, where the main file is in the root if there are no other code files.

Reply:
I'd say it brings bad DX, you will end up with new files in the root directory after compilation (at least the `.js` and `.d.ts`, but probably even more files, there can be source maps for both of those generated files).

All the crawlee templates are using `src` folder, both JS and TS, so if you want consistency, why are you removing the folder it in the first place?

cc @mnmkng @jbartadev, I saw similar discussion in the other PR

Reply:
Do you mean the created files after running `tsc`? Those end up in a dist folder so it should not really affect the DX no?

Reply:
I trust @B4nan on this. It seems the marginal simplification of the template thanks to skipping the `src` folder can actually bring important problems down the road. Looks like keeping the folder will actually make it easier for users.

Reply:
Fair enough thx for explanation :)